models/pianocktail_cnn.py

```diff
@@ -18,24 +18,33 @@ def __init__(self, name=config.ModelEnum.PIANOCKTAIL_CNN.value, subspectrogram_p
 
         self.pooling1 = AveragePooling2D(pool_size=2,name=f"{name}_pool1")
 
+        self.dropout1 = Dropout(0.1,name=f"{name}_dropout1")
```

Comment:
Not sure about the hard coded 0.1, this usually needs to be cross validated

Reply:
Sure, it was just a try since we don't have cross validation

Reply:
Or at least validated, you get the point :smile: